Scheduler/observing_blocks_maintel/BLOCK-T299.json

```diff
@@ -0,0 +1,48 @@
+{
+    "name": "BLOCK-T299",
+    "program": "BLOCK-T299",
+    "constraints": [],
+    "scripts": [
+          {
+            "name": "maintel/point_azel.py",
+            "standard": true,
+            "parameters": {
+                "az": 295.713,
+                "el": 29.095,
+                "target_name": "EUTELSAT_117W_A",
+                "reason": "SITCOM-1284",
+                "wait_dome": true
+            }
+        },
+        {
+            "name": "run_command.py",
+            "standard": true,
+            "parameters": {
+                "component": "MTPtg",
+                "cmd": "startTracking"
+            }
+        },
+        {
+            "name": "maintel/take_image_comcam.py",
+            "standard": true,
+            "parameters": {
+                "image_type": "ACQ",
```

Comment:
You should add the "program": $program keyword here so that it gets propagated to the image headers.

Reply:
Done.